src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="block start--day-wed month--length-31">
```

Comment:
Those `start--day-wed month--length-31` are incorrect. It should be `calendar--start-day--` and `calendar--month-length--`